### PR TITLE
yocto: labs: Add missing nfsvers configuration

### PR DESCRIPTION
**Pull request by dagmcr**
Signed-off-by: Daniel Gomez <dagmcr@gmail.com>

### CODE REVIEW

File: labs/yocto-advanced-configuration/yocto-advanced-configuration.tex [hunk 1]
```diff
@@ -17,7 +17,8 @@ \section{Set up the Ethernet communication and NFS on the board}
 First we need to set the kernel boot arguments U-Boot will pass to the
 Linux kernel at boot time:
 \begin{verbatim}
-setenv bootargs 'console=ttyS0,115200 root=/dev/nfs rw nfsroot=192.168.0.1:/nfs
+setenv bootargs 'console=ttyS0,115200 root=/dev/nfs rw
+  nfsroot=192.168.0.1:/nfs,nfsvers=3
```
Comment (alexandrebelloni) [inline labs/yocto-advanced-configuration/yocto-advanced-configuration.tex]: The tcp option should probably be added too because distributions are starting to remove udp support.

File: labs/yocto-advanced-configuration/yocto-advanced-configuration.tex [hunk 2]
```diff
@@ -172,7 +173,7 @@ \section{Going further}
 
 \begin{enumerate}
 
-\item Install a TFTP server (package \code{tftpd-hpa}) on your system. 
+\item Install a TFTP server (package \code{tftpd-hpa}) on your system.
```
Comment (alexandrebelloni) [inline labs/yocto-advanced-configuration/yocto-advanced-configuration.tex]: This is an unrelated change.

File: labs/yocto-advanced-configuration/yocto-advanced-configuration.tex (hunk 1 above)
Comment (michaelopdenacker) [reply labs/yocto-advanced-configuration/yocto-advanced-configuration.tex]: Then, we should do it for all our labs, through a separate commit.

File: labs/yocto-advanced-configuration/yocto-advanced-configuration.tex (hunk 2 above)
Comment (michaelopdenacker) [reply labs/yocto-advanced-configuration/yocto-advanced-configuration.tex]: I'm ready to take it anyway, as it's a minor cleanup one.

File: labs/yocto-advanced-configuration/yocto-advanced-configuration.tex (hunk 1 above)
Comment (michaelopdenacker) [reply labs/yocto-advanced-configuration/yocto-advanced-configuration.tex]: Agreeing with the "nfsvers=3" variant

**Comment by michaelopdenacker** [comment]
Merged. Thanks!
